ingestion/tests/unit/helpers_test.py

```diff
@@ -12,53 +12,54 @@
 from metadata.generated.schema.api.data.createTable import CreateTableEntityRequest
 from metadata.generated.schema.api.services.createDatabaseService import CreateDatabaseServiceEntityRequest
 from metadata.generated.schema.entity.data.table import Column
 from metadata.generated.schema.type.entityReference import EntityReference
 from metadata.ingestion.ometa.client import REST, APIError
+from metadata.ingestion.ometa.openmetadata_rest import OpenMetadataAPIClient
+from metadata.ingestion.ometa.openmetadata_rest import MetadataServerConfig
+from metadata.generated.schema.api.services.createDashboardService import CreateDashboardServiceEntityRequest
+from metadata.generated.schema.api.services.createMessagingService import CreateMessagingServiceEntityRequest
 
 
 class RestTest(TestCase):
 
-    @classmethod
-    def setUpClass(cls) -> None:
-        cls._client = REST("http://localhost:8585/api", 'test', 'none')
-
-    @classmethod
-    def tearDownClass(cls) -> None:
-        pass
+    file_path = 'tests/unit/mysql_test.json'
+    with open(file_path) as ingestionFile:
+        ingestionData = ingestionFile.read()
+    client_config = json.loads(ingestionData).get("metadata_server")
+    config = client_config.get("config", {})
+    metadata_config1 = MetadataServerConfig.parse_obj(config)
+    _client = OpenMetadataAPIClient(metadata_config1)
 
-    def test_get_and_delete_service(self):
-        mysql_service = self._client.get_database_service('local_mysql')
-        self.assertEqual(mysql_service.name, 'local_mysql')
-        self._client.delete_database_service(mysql_service.id.__root__)
-        self.assertRaises(APIError, self._client.get_database_service_by_id, mysql_service.id.__root__)
-
-    def test_create_service(self):
-        data = {'jdbc': {'connectionUrl': 'mysql://localhost/catalog_db', 'driverClass': 'jdbc'},
-                'name': 'local_mysql',
+    def test_1_create_service(self):
+        data = {'jdbc': {'connectionUrl': 'mysql://localhost/openmetadata_db', 'driverClass': 'jdbc'},
+                'name': 'local_mysql_test',
+                'serviceType': "MySQL",
                 'description': 'local mysql env'}
         create_mysql_service = CreateDatabaseServiceEntityRequest(**data)
         mysql_service = self._client.create_database_service(create_mysql_service)
-        print(mysql_service)
         self.assertEqual(mysql_service.name, create_mysql_service.name)
 
-    def test_get_service_by_id(self):
-        mysql_service = self._client.get_database_service('local_mysql')
-        self.assertEqual(mysql_service.name, 'local_mysql')
+    def test_2_get_service(self):
+        mysql_service = self._client.get_database_service('local_mysql_test')
+        self.assertEqual(mysql_service.name, 'local_mysql_test')
+
+    def test_3_get_service_by_id(self):
+        mysql_service = self._client.get_database_service('local_mysql_test')
+        self.assertEqual(mysql_service.name, 'local_mysql_test')
```

Comment:
Can you keep assertEqual one per testcase?

Reply:
Addressed in the latest commit

Reply:
@parthp2107 Few tests aren't passing in helpers_test.py
can you take a look at it?

Reply:
It is now fixed. Addressed in the latest commit